pandas/core/indexing.py

```diff
@@ -1733,6 +1734,37 @@ def _getitem_axis(self, key, axis: int):
         self._validate_key(key, axis)
         return self._get_label(key, axis=axis)
 
+    def _ensure_listlike_indexer(self, key: Iterable, axis: int):
```

Comment:
this by-definition should always be on axis=1, yes? i would remove that as an argument and/or be very explicit about this.

Reply:
> this by-definition should always be on axis=1, yes?

Yes, `axis` is always 1.